<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Grid_Dialog</class>
 <widget class="QDialog" name="Grid_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>271</width>
    <height>380</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>271</width>
    <height>380</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>271</width>
    <height>380</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Grid options</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <widget class="QPushButton" name="_2">
   <property name="geometry">
    <rect>
     <x>148</x>
     <y>330</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="_1">
   <property name="geometry">
    <rect>
     <x>58</x>
     <y>330</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>75</width>
     <height>23</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>75</width>
     <height>23</height>
    </size>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>20</y>
     <width>107</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>X spacing</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>50</y>
     <width>109</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Y spacing</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>80</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>X reference grid</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>110</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Y reference grid</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>140</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Angle</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="_105">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>26</x>
     <y>174</y>
     <width>205</width>
     <height>21</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string>Snap to grid</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="tristate">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="_107">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>28</x>
     <y>214</y>
     <width>203</width>
     <height>21</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string>Display reference grid only</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="tristate">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="_106">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>28</x>
     <y>194</y>
     <width>203</width>
     <height>21</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string>Display grid</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="tristate">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="RCDoubleSpinBox" name="_100">
   <property name="geometry">
    <rect>
     <x>134</x>
     <y>18</y>
     <width>115</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <double>100000000.000000000000000</double>
   </property>
  </widget>
  <widget class="RCDoubleSpinBox" name="_101">
   <property name="geometry">
    <rect>
     <x>134</x>
     <y>48</y>
     <width>115</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <double>100000000.000000000000000</double>
   </property>
  </widget>
  <widget class="RCDoubleSpinBox" name="_104">
   <property name="geometry">
    <rect>
     <x>134</x>
     <y>138</y>
     <width>115</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>0.000000000000000</double>
   </property>
   <property name="maximum">
    <double>360.000000000000000</double>
   </property>
  </widget>
  <widget class="RCIntegerSpinBox" name="_102">
   <property name="geometry">
    <rect>
     <x>134</x>
     <y>78</y>
     <width>115</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>1000000000</number>
   </property>
  </widget>
  <widget class="RCIntegerSpinBox" name="_103">
   <property name="geometry">
    <rect>
     <x>134</x>
     <y>108</y>
     <width>115</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>1000000000</number>
   </property>
  </widget>
  <widget class="QCheckBox" name="_108">
   <property name="geometry">
    <rect>
     <x>32</x>
     <y>234</y>
     <width>199</width>
     <height>20</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string>Display grid as lines</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="_109">
   <property name="geometry">
    <rect>
     <x>38</x>
     <y>254</y>
     <width>193</width>
     <height>20</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string>Use custom grid colour</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="_111">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>196</x>
     <y>282</y>
     <width>32</width>
     <height>16</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="backgroundBrush">
    <brush brushstyle="NoBrush">
     <color alpha="255">
      <red>0</red>
      <green>0</green>
      <blue>0</blue>
     </color>
    </brush>
   </property>
   <property name="sceneRect">
    <rectf>
     <x>0.000000000000000</x>
     <y>0.000000000000000</y>
     <width>32.000000000000000</width>
     <height>16.000000000000000</height>
    </rectf>
   </property>
   <property name="viewportUpdateMode">
    <enum>QGraphicsView::FullViewportUpdate</enum>
   </property>
  </widget>
  <widget class="QLabel" name="_110">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>34</x>
     <y>282</y>
     <width>157</width>
     <height>16</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Grid colour</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="_112">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>44</x>
     <y>306</y>
     <width>147</width>
     <height>16</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Grid colour</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QGraphicsView" name="_113">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>196</x>
     <y>306</y>
     <width>32</width>
     <height>16</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="backgroundBrush">
    <brush brushstyle="NoBrush">
     <color alpha="255">
      <red>0</red>
      <green>0</green>
      <blue>0</blue>
     </color>
    </brush>
   </property>
   <property name="sceneRect">
    <rectf>
     <x>0.000000000000000</x>
     <y>0.000000000000000</y>
     <width>32.000000000000000</width>
     <height>16.000000000000000</height>
    </rectf>
   </property>
   <property name="viewportUpdateMode">
    <enum>QGraphicsView::FullViewportUpdate</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="_1110">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>278</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../RealCAD.qrc">
     <normaloff>:/images/PROPERTIES-colours.svg</normaloff>:/images/PROPERTIES-colours.svg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="_1130">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>302</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../RealCAD.qrc">
     <normaloff>:/images/PROPERTIES-colours.svg</normaloff>:/images/PROPERTIES-colours.svg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RCDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>../RealCADQt/source/rcdoublespinbox.h</header>
  </customwidget>
  <customwidget>
   <class>RCIntegerSpinBox</class>
   <extends>QSpinBox</extends>
   <header>../RealCADQt/source/rcintegerspinbox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>_100</tabstop>
  <tabstop>_101</tabstop>
  <tabstop>_102</tabstop>
  <tabstop>_103</tabstop>
  <tabstop>_104</tabstop>
  <tabstop>_105</tabstop>
  <tabstop>_106</tabstop>
  <tabstop>_107</tabstop>
  <tabstop>_108</tabstop>
  <tabstop>_109</tabstop>
  <tabstop>_111</tabstop>
  <tabstop>_1110</tabstop>
  <tabstop>_113</tabstop>
  <tabstop>_1130</tabstop>
  <tabstop>_1</tabstop>
  <tabstop>_2</tabstop>
 </tabstops>
 <resources>
  <include location="../RealCAD.qrc"/>
 </resources>
 <connections/>
</ui>
